Makefile

```diff
@@ -15,6 +15,7 @@ export PATH := $(GOBIN):$(PATH):$(BIN_DIR)
 GOPATH = $(CURDIR)/.gopath
 ORG_PATH = github.com/k8snetworkplumbingwg
 PACKAGE = ovs-cni
+OCI_BIN ?= $(shell if hash podman 2>/dev/null; then echo podman; elif hash docker 2>/dev/null; then echo docker; fi)
```

Comment:
This assumes that in case podman / docker exists it is also the running service
later on we can improve it if needed